http-gateway/service/config.go

```diff
@@ -1,38 +1,34 @@
 package service
 
 import (
-	"fmt"
-	"reflect"
+	"time"
 
+	"github.com/plgd-dev/kit/config"
 	"github.com/plgd-dev/kit/security/certManager"
-
-	"gopkg.in/yaml.v2"
 )
 
 // Config represent application configuration
 type Config struct {
-	Address                  string             `envconfig:"ADDRESS" default:"0.0.0.0:7000"`
+	Address                  string             `envconfig:"ADDRESS"`
 	Listen                   certManager.Config `envconfig:"LISTEN"`
 	Dial                     certManager.Config `envconfig:"DIAL"`
 	JwksURL                  string             `envconfig:"JWKS_URL"`
-	ResourceDirectoryAddr    string             `envconfig:"RESOURCE_DIRECTORY_ADDRESS"  default:"127.0.0.1:9100"`
-	CertificateAuthorityAddr string             `envconfig:"CERTIFICATE_AUTHORITY_ADDRESS"  default:""`
+	ResourceDirectoryAddr    string             `envconfig:"RESOURCE_DIRECTORY_ADDRESS"`
+	CertificateAuthorityAddr string             `envconfig:"CERTIFICATE_AUTHORITY_ADDRESS"`
+	WebSocketReadLimit       int64              `envconfig:"WEBSOCKET_READ_LIMIT"`
```

Comment:
This MR contains also changes which are not relevant for DELETE command support.

Reply:
I know ...